src/core/src/main/java/org/apache/jmeter/control/LoopController.java

```diff
@@ -81,19 +81,19 @@ public int getLoops() {
         // Evaluation occurs when nbLoops is not yet evaluated
         // or when nbLoops is equal to special value INFINITE_LOOP_COUNT
         if (nbLoops==null || // No evaluated yet
-                nbLoops.intValue()==0 || // Last iteration led to nbLoops == 0,
+                nbLoops ==0 || // Last iteration led to nbLoops == 0,
                                          // in this case as resetLoopCount will not be called,
                                          // it leads to no further evaluations if we don't evaluate, see BUG 56276
```

Comment:
Is it worth fixing this indentation (or moving it to a form which doesn't require constant manual formatting?)

Reply:
It would probably be nice to use consistent spacing around `==` provided Spotless or something like that can auto-format the code.

Reply:
Sorry, I was referring to the block of `//` comments, but now you mention it, the spacing should at least be consistently wrong or correct ;)

Reply:
Ah, I see. As discussed earlier, there's no point in maintaining horizontal alignment.
Frankly speaking, I don't like "line comments within expression", and it would probably look better if the comment is placed before `if` or if `nbLoops ==0` was extracted to a separate method.

I'm not sure it makes sense to manually inspect this change and fix spacing around `==` because it would take tim and produce no value.
I would rather spend the time on implementing Async HTTP client rather than dealing with spaces.